cryptofeed/rest/kraken.py

```diff
@@ -160,7 +160,8 @@ def trades(self, symbol: str, start=None, end=None, retry=None, retry_wait=10):
             if not end:
                 end = pd.Timestamp.utcnow()
             for data in self._historical_trades(symbol, start, end, retry, retry_wait):
-                yield list(map(lambda x: self._trade_normalization(x, symbol), data['result'][next(iter(data['result']))]))
+                yield list(
+                    map(lambda x: self._trade_normalization(x, symbol), data['result'][next(iter(data['result']))]))
```

Comment:
looks like just a formatting change, please undo

Reply:
this is now resolved